### PR TITLE
no need to use setuptools_cython anymore

### PR DESCRIPTION
**Pull request by pypeng**
```
cython is working fine
```

### CODE REVIEW

**Comment by pypeng** [comment]
need this: https://github.com/douban/greenify/pull/14

**Comment by windreamer** [comment]
@youngsofun pls review

**Comment by mckelvin** [comment]
It works fine on latest version of `setuptools`. So what's the minimum required version of `setuptools`? since old versions may break. And it would be helpful to warning something if `setuptools.__version__` is not expected.

**Comment by windreamer** [comment]
@pypeng we should ensure `setuptools >= 0.18.0`, I think.  @mckelvin

**Comment by youngsofun** [comment]
@pypeng  what's your opinion  about setuptools version?

**Comment by pypeng** [comment]
let me find the minimum version for setuptools.

**Comment by windreamer** [comment]
@pypeng 
cf: http://setuptools.readthedocs.io/en/latest/history.html#id174

File: setup.py [hunk 1]
```diff
@@ -71,7 +64,7 @@ def run_tests(self):
         "Topic :: Software Development :: Libraries"
     ],
     # Support for the basestring type is new in Cython 0.20.
-    setup_requires=["setuptools_cython", "Cython >= 0.20"],
+    install_requires=["Cython >= 0.20", "setuptools >= 18.0"],
```
Comment (windreamer) [inline setup.py]: seems this does not work for setuptools. maybe we should add an assert to check setuptools version?

**Comment by pypeng** [comment]
yes, setuptools==18.0 is the minimum required version. i've updated setup.py.

I also changed setup_requires to install_requires, which is required by pip: https://pip.pypa.io/en/stable/user_guide/?highlight=install_requires

File: setup.py (hunk 1 above)
Comment (windreamer) [inline setup.py]: OK, `setup_requires` seems working, `install_requires` doesn't ...

**Comment by windreamer** [comment]
@pypeng I think just a note in `README` is enough... all these hacks looks weird and confuse.

**Comment by windreamer** [comment]
LGTM

**Pull request merged**